geomesa-hbase/geomesa-hbase-datastore/src/main/scala/org/locationtech/geomesa/hbase/coprocessor/KryoLazyDensityCoprocessor.scala

```diff
@@ -90,7 +89,6 @@ class KryoLazyDensityCoprocessor extends KryoLazyDensityService with Coprocessor
 
       scanList.foreach(scan => {
         scan.setFilter(filterList)
-        // TODO: Explore use of MultiRangeFilter
```

Comment:
it's a different logic branch from the coprocessors - the current serialization of scans doesn't work with the mrrf so would have to be updated.